VersionMiner/Services/CSharpProjFileService.cs

```diff
@@ -0,0 +1,97 @@
+﻿// <copyright file="CSharpProjFileService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+using VersionMiner.Guards;
+
+namespace VersionMiner.Services;
+
+/// <summary>
+/// Performs operations on data in CSharp project files.
```

Comment:
Should it be C#?   If so, refactor all to C#.

Reply:
The code docs can be `C#` instead of `CSharp`.  But when it comes to the data type names of the code itself, using the hash symbol `#` is not aloud and is an illegal character for the C# compiler.  That is why the class name is spelled out as `CSharp`.

🤓